cli/src/services/document.ts

```diff
@@ -115,28 +129,51 @@ export default class Document {
     const query = [
       ['document_path', documentPath],
       ['document_format', this.config.format]
-    ]
-      .map(([name, value]) => `${name}=${value}`)
-      .join('&');
+    ];
 
-    const url = `${this.apiUrl}/jipt-export?${query}`;
+    const url = `${this.apiUrl}/jipt-export?${this.encodeQuery(query)}`;
     const response = await fetch(url, {
       headers: this.authorizationHeader()
     });
 
     return this.writeResponseToFile(response, file);
   }
 
+  private encodeQuery(params: string[][]) {
+    return params.map(([name, value]) => `${name}=${value}`).join('&');
```

Comment:
We should use `encodeURIComponent` here.

```suggestion
    return params.map(([name, value]) => `${name}=${encodeURIComponent(value)}`).join('&');
```